docker/Dockerfile

```diff
@@ -1,10 +1,13 @@
+# syntax=docker/dockerfile:1.6
 ARG RUST_VERSION=1.72.1
 ARG CARGO_BUILD_FEATURES=default
 ARG RUST_RELEASE_MODE=debug
+
 ARG AMD_BUILDER_IMAGE=rust:${RUST_VERSION}
-ARG ARM_BUILDER_IMAGE=blackdex/rust-musl:aarch64-musl-stable-${RUST_VERSION}-openssl3
+ARG ARM_BUILDER_IMAGE="ghcr.io/raskyld/aarch64-lemmy-linux-gnu:v0.1.0"
```

Comment:
Isnt it possible to move it into this dockerfile using multistage build?

https://docs.docker.com/build/building/multi-stage/

Reply:
I am not sure I understand but I guess you mean to extract the docker image building the toolchain and putting it there?

I put it in another repository because building a cross toolchain is a really heavy process, on free github runner it **took an hour to build**: https://github.com/raskyld/lemmy-cross-toolchains/actions/runs/6792416632.

Even during development, I used a [Scaleway POP2-HC](https://www.scaleway.com/en/docs/compute/instances/reference-content/workload-optimized/) (8 dedicated vCPU and 16 Go RAM) and it still took something between 20 to 30 minutes :/ So I would really recommend to avoid letting   this is the Dockerfile of the Lemmy repository for two reason:

1. It's easy to mess with the docker cache and accidentally re-trigger a full cross toolchain build.
2. Even though *buildroot* can make use of *ccache*, it's non re-liable and it needs manual re-full-build trigger.
3. It would make the dockerfile heavier and harder to read :/

Reply:
If for governance and ownership reasons you would prefer to migrate it to the LemmyNet orga, migration is indeed possible! We can even let it on the mono-repo but I would still recommend letting it be an independent image.

Reply:
Compile time is a good argument. I dont mind if you maintain the image as long as it doesnt break.